test_common/resources/ConsumerGroups.view.tsx

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
I think a generator function might be better instead of duplication - `(componentName) => (): ReactElement => {
  return <div>{componentName}</div>;
}`